src/components/alert/alert.stories.svelte

```diff
@@ -91,8 +91,10 @@
   component={Alert}
   argTypes={{
     modes: { table: { disable: true } },
+    // @ts-ignore
```

Comment:
@fallaciousreasoning, I couldn't do `options: modes as any` since it was flagging `as` as an unknown token. I believe this is because TS isn't currently supported within the HTML portion of Svelte components? That's a bit of a guess, though.

Anyhow, figuring `@ts-ignore` works just as well? :-D

Reply:
I think that's right, about `Typescript` not being supported in the HTML bit of `Svelte` components.

Hey, I just had another idea - can you pass through a copy of the props?

`modes={[...modes]}` or something?

Reply:
That would probably have worked!

However, it appears that it's no longer an issue? Maybe because of a still more recent Storybook update I applied?